### ISSUE TITLE
Issues with tests on `TestUpdates` branch

### ISSUE THREAD

**Issue by yasz24** [jeremyeng/cs4500_project#22]
We did our implementation based on your previous API with two separate string and object queues. 
I would like to open a pull request to update the tests on the `TestUpdates` branch to address some of the problems with you old test suite:
1. Segfault when `delete string3`.
2. A bunch of redeclaration issues for `popped` on lines 84-96.

Would it be possible to add me as a contributor, so I can open a pull request to update that branch? 
If not would it be possible to furnish those fixes? I believe they have already been fixed for the version  on master.

**Comment by njhofmann** [comment]
Yes I do believe those issues were generously fixed by another group, but if you've already implemented the both APIs - should you not be able to just run the tests we have on master? Is there any particular reason why you need to use the old test suite? Is is strictly inferior to the one on master, at least for the default Object Queue.

I would just recommend sticking with the current spec as much as you can. You don't need to remove StringQueue or any other tests you've written, you can just say your interpretations of the spec differed from ours so you decided to write another class specifically for Strings.

However you haven't tried already, you should be able to make a PR to this repo yourself by 1.) forking the repo, 2.) making the changes you want on your local machine (on non-master branch preferably), 3.) pushing the branch to your forked remote repo, and 4.) putting in a PR from Github.

**Comment by yasz24** [comment]
No, because they instantiate an object of type `Queue`, but we implemented `StringQueue` and `ObjectQueue`

**Comment by njhofmann** [comment]
Can you not just rename the ObjectQueue to Queue to be inline with the spec?

**Comment by yasz24** [comment]
Uh, okay sure, will try that

**Comment by njhofmann** [comment]
Well lmk if that doesn't for some reason, I imagine it should because your ObjectQueue class has the same API as the Queue class correct?
